src/lib/utils/APIConstants.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Separating API endpoints into their own constants file
+ */
+
+import { getEndpoint, BackendDestinationEnum } from './functions/getEndpoint'
+
+export const BACKEND_ENDPOINT = `${getEndpoint(
+  BackendDestinationEnum.REPO_ENDPOINT,
+)}`
+
+export const REPO = `/repo/v1`
+export const AUTH = `/auth/v1`
+export const FILE = `/file/v1`
+
+export const ENTITY = `${REPO}/entity`
+export const ENTITY_ID = (id: string | number) => `${REPO}/entity/${id}`
```

Comment:
Thoughts on using functions to reuse template literals?

Reply:
Ok with me (although I'm not sure about the style to have all letters capitalized for this function)

Reply:
What do others on the web team think?